### PR TITLE
display the board on the page

### PR DESCRIPTION
**Pull request by poorvarathore05**

### CODE REVIEW

**Comment by montrealist** [comment]
Hi Poorva,

Can you please update the `.gitignore` file to exclude the `__pycache__` directory? The files from this directory should not go into source control.

Also please add a `README.md` file with instructions on how to run the app locally.

**Comment by poorvarathore05** [comment]
Sure, I will add that file in .gitignore and will update.

File: flask_boggle/app.py [hunk 1]
```diff
@@ -13,4 +14,16 @@ def start_page():
 @app.route('/start')
 def boggle_page():
     make_board = boggle_game.make_board()
-    return render_template('table.html',board = make_board)
+    session['board'] = make_board
+    high_score = session.get('high_score',0)
+    nplays = session.get('nplayes',0)
```
Comment (montrealist) [inline flask_boggle/app.py]: Do you have a typo here? `nplays` vs `nplayes`

File: flask_boggle/templates/base.html [hunk 2]
```diff
@@ -10,9 +10,16 @@
...
+    <link rel="stylesheet" href="/static/boggle.css" />
     <title>Boggle</title>
   </head>
   <body>
     {%block content%} {% endblock %}
+    <script src="http://unpkg.com/jquery"></script>
+    <script src="https://unpkg.com/axios@0.19.0/dist/axios.js"></script>
+    <script src="/static/boggle.js"></script>
+    <script>
+      let game = new BoggleGame("boggle", 60);
```
Comment (montrealist) [inline flask_boggle/templates/base.html:22]: This can be const if you’re not going to modify it.

File: flask_boggle/app.py (hunk 1 above)
Comment (poorvarathore05) [reply flask_boggle/app.py]: Yeah, thank you to give the attention on that

**Pull request merged**